python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -161,6 +161,17 @@ def l2_pool2d_pattern():
         pattern = is_op("sqrt")(pattern)
         return pattern
 
+    def add_pattern():
+        """Create an add pattern.
+
+        Returns
+        -------
+        pattern : dataflow_pattern.AltPattern
+            Denotes the convolution pattern.
+        """
+        pattern = is_op("add")(wildcard(), is_constant(), is_constant())
```

Comment:
Why add has 3 arguments? Did I miss something?

Reply:
I'm not sure we need to define a pattern for add? as it is a single op we shouldn't need to wrap it within a composite function

Reply:
removed